quality/errorprone/android.gradle

```diff
@@ -6,8 +6,8 @@ dependencies {
     errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
   }
   errorprone 'com.google.errorprone:error_prone_core:2.3.2'
-  annotationProcessor 'com.uber.nullaway:nullaway:0.6.0'
-  testAnnotationProcessor 'com.uber.nullaway:nullaway:0.6.0'
```

Comment:
looks like 0.6.5 is out since yesterday xD
https://github.com/uber/NullAway/releases

- [ ] update to 0.6.5

Reply:
Updated